scripts/bls/eigen/pause.sh

```diff
@@ -16,14 +16,16 @@ parse_args "$@"
 
 # Check required parameters with defaults
 check_param "DEPLOY_ENV" "${DEPLOY_ENV:-LOCAL}"
-check_param "SLASHING_REGISTRY_COORDINATOR_ADDRESS" "${SLASHING_REGISTRY_COORDINATOR_ADDRESS:-$(jq -r '.addresses.registryCoordinator' "$HOME/.nodes/avs_deploy.json")}"
+DEFAULT_REGISTRY_ADDRESS=$(jq -r '.addresses.registryCoordinator' "$HOME/.nodes/avs_deploy.json" || true)
+check_param "REGISTRY_ADDRESS" "${REGISTRY_ADDRESS:-$DEFAULT_REGISTRY_ADDRESS}"
 
 # Set up environment based on DEPLOY_ENV
 setup_environment
 
-# Read the deployer private key from file
-deployer_private_key=$(load_deployment_data "$HOME/.nodes/deployer")
-deployer_address=$(cast wallet address "$deployer_private_key")
+# Read the deployer private key
+deployer_private_key=$(load_deployment_data "$HOME/.nodes/deployer" || true)
+check_param "FUNDED_KEY" "${FUNDED_KEY:-$deployer_private_key}"
```

Comment:
Would it make more sense for this arg to just be DEPLOYER_PK instead of FUNDED_KEY?